Dockerfile

```diff
@@ -37,7 +37,9 @@ RUN apt-get update \
     python3-pip \
     python3 \
     python \
-    zlib1g-dev
+    zlib1g-dev \
+  && apt-get clean \
+  && apt-get autoremove
```

Comment:
You can go one step further and delete the `apt` caches like this:

https://github.com/ElementsProject/lightning/blob/ab0fa7a1bd156574dd2568bfdbe70255426e928d/contrib/Dockerfile.builder#L38

Reply:
It looks shiny!
fixed